middleware/auth.js

```diff
@@ -0,0 +1,23 @@
+const Token = require("../models/token");
+
+module.exports = () => {
+    return async (req, res, next) => {
+        // console.log(req.headers.authorization);
+        const authToken = req.headers.authorization;
```

Comment:
What if req.headers.authorization isn't present. 
I tried many times testing where I forgot to include this and had to restart the sever. ;_;

Reply:
What else can we do here? Can you suggest something?

Reply:
You can use `optional chaining`, to ensure that if `headers` is not present, then we do not get any error `Cannot read property authorization of undefined`.

Reply:
Okay!